guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,8 +4,42 @@
 [id="katello-headline-features"]
 == Headline Features
 
+=== Verify Content Checksum on container repositories using Hammer
+Added option '--validate-contents true/false' to "hammer repository synchronize" command.
```

Comment:
Please take this as a suggestion on how to write headline features. I'm taking this one because it's the first one, but you can ask yourself the same questions for other headline features.

When I read this line I have more questions than answers. For example, what was the previous behavior? Was it never validating or always? What is the effect if I pass one or the other? Why would I choose to not verify checksums, because I'd think that's always a good thing.

The preview is rendered. Please compare the Foreman headline features (https://theforeman-foreman-documentation-preview-pr-2481.surge.sh/3.8/Release_Notes/index-katello.html#foreman-headline-features) with the Katello ones. With those I've tried to compare the previous state to the new state and explain why the new state is better. I won't claim to be very good at it, but that's at least the thought process behind it.

Reply:
Yeah @damoore044 this one is bit confusing -- context is that we've always had this `--validate-contents` flag, but it didn't work for container repositories (and some others) due to some historical reason. So, we just "unlocked" `--validate-contents` for all repo types.  It was all a backend change, not even a change in hammer.

Also, `--validate-contents` actually triggers a different task in Pulp that's not technically even a repository sync. Back in Pulp 2, a validate content sync would also sync the repository (if I remember right). But, in Pulp 3, validating contents is a specific task that just checks for proper checksums.